packages/bitcoin-dlc-provider/lib/BitcoinDlcProvider.ts

```diff
@@ -1118,6 +1118,9 @@ Payout Group not found',
       _dlcTxs,
     });
 
+    if (isOfferer === undefined)
```

Comment:
Can be changed to `if (!isOfferer)`

Reply:
This might not work since a `false` boolean can be passed in as well

Reply:
Good point! In that case LGTM 👍